### PR TITLE
Feat: reasonable default arguments

### LINKED ISSUE #11: Match lodash defaults
Some of the defaults for lodash are a bit different from underdash.
Example:
```
lodash.chunk([1,2,3,4]) // := [ [ 1 ], [ 2 ], [ 3 ], [ 4 ] ]
// vs. 
chunks([1,2,3,4]) // := [ [] ]
```

If you are accepting PR's I'll sign the CLA and send some PR's to put default values of `size = 1` for that function and `gt = (a, b) => a > b` for `min` and `max` and `e => e` for `intersection` and `unique`.

### PR DESCRIPTION
**Pull request by josher19**
Fixes #11 for min, max, and chunks.

### CODE REVIEW

File: f/chunks.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-function chunks(arr, size) {
+function chunks(arr, size = 1) {
```
Comment (surma) [inline f/chunks.js:18]: I disagree on giving `chunks` a default.

**Review by surma — CHANGES_REQUESTED** [review]
I like the idea of reasonable defaults! Happy to merge this, but I don’t think there is such a thing as a reasonable default for `chunks` specifically.

File: f/chunks.js (hunk 1 above)
Comment (josher19) [reply f/chunks.js:18]: Okay, I can revert that but then it default to 0:
`chunks([1,2,3,4])` becomes `[ [] ]`

Comment (josher19) [reply f/chunks.js]: Okay, reverted and fixed broken test. 
Also used identify function as default for `unique`, `uniqueLazy` and `intersection`.

**Pull request merged**